content/ux-strat-bootcamp/week-one/tuesday/user-flows/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: User Flows
+content_type: project
+ready: true
+---
+
+User flow or UX flow is a visual representation of a specific route that a user might take through a website or app in order to achieve a goal. The route starts at a specific point of entry and covers all the steps the user must take to reach a certain outcome. There is often more than one route a user could follow to complete the task. When thinking about user flows, it’s important to consider all the ways users could potentially navigate through your site or app so that you can identify key pathways (generally those that are the most simple and direct).
+
+**Task (1hr)**
+Design a user flow diagram for your user navigating through an app or website to enhance their commute experience. Think through what you discovered with their unique aspects and needs of their daily commute in your user journey. Tip: do some research on commute apps to get ideas on how you could design your user flow diagram.  
+
+**How to do this? (1hr)** 
+You must first figure out the objective of the user flow is. User flows that fail to satisfy these objectives don’t offer value—to you or your end users.
+Record your user flows in a simple bullet-point list first and use the correct notation. Do some research on user flow notations - remember, Google is your friend. 
+Figure out what the most critical UX flows are.
+Map out the flow from the user’s perspective. Take into consideration your user’s pain points, level of expertise, and preferences.
+
+**Additional resources (1hr)**
+Visit [this link](https://www.figma.com/community/file/830510773896272856) to help you complete your user flow diagram.
+Check out this link on [Lucid Chart](https://www.lucidchart.com/blog/how-to-make-a-user-flow-diagram) to potentially help you
+[A beginner’s guide to user flows](https://careerfoundry.com/en/blog/ux-design/what-are-user-flows/#:~:text=User%20flows%2C%20UX%20flows%2C%20or,through%20to%20the%20final%20interaction.)
+You’re welcome to do your own additional research! 
+
+**Expected outcomes/Deliverables**
+A simple user flow diagram.
```

Comment:
How should the user submit their work? Should they submit a link to a file? If that is the case it is worth being very explicit about what is expected.

Reply:
If you want them to submit a link then the project submission type should be set to `link`